assets/src/models/routeTab.ts

```diff
@@ -75,6 +75,31 @@ export const isEditedPreset = (routeTab: RouteTab): boolean =>
 export const isOpenTab = (routeTab: RouteTab): boolean =>
   routeTab.ordering !== undefined
 
+export const tabName = (routeTab: RouteTab): string =>
+  routeTab?.presetName || "Untitled"
+
+export const currentTabName = (routeTabs: RouteTab[]): string => {
+  const currentTab = currentRouteTab(routeTabs)
+  if (currentTab !== undefined) return tabName(currentTab)
+  return "Untitled"
+}
+
+export const toTitleCase = (str: string): string => {
+  return str.replace(
+    /\w\S*/g,
+    (txt: string): string =>
+      txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase()
+  )
+}
+
+export const pageOrTabName = (routeTabs: RouteTab[]): string => {
+  let tabName = "Skate"
+  if (location.pathname === "/") tabName = currentTabName(routeTabs)
+  else
+    tabName = toTitleCase(location.pathname.replace("/", "").replace("-", " "))
+  return tabName
+}
```

Comment:
A couple of things on these functions:
1. It's probably better to use the `useLocation()` hook from `react-router-dom` instead of referencing so that the component rerenders if the location changes, which might be relevant with how React router does things internally. Note that since you can't call hooks from functions that aren't components this means you'd need to take the location as an argument to this function and put the `useLocation` call in the relevant component(s).
2. I'm not sure that the logic about the page title really belongs in `routeTab.ts` - it might make sense to just put these in the same file as the mobile top nav component as helper functions if that's the only place in the app that needs this logic.

Reply:
fixed in 3bac2c8